pkg/resource/db_cluster_parameter_group/delta.go

```diff
@@ -63,8 +63,12 @@ func newResourceDelta(
 			delta.Add("Spec.Name", a.ko.Spec.Name, b.ko.Spec.Name)
 		}
 	}
-	if !reflect.DeepEqual(a.ko.Spec.Parameters, b.ko.Spec.Parameters) {
-		delta.Add("Spec.Parameters", a.ko.Spec.Parameters, b.ko.Spec.Parameters)
+	if ackcompare.HasNilDifference(a.ko.Spec.ParameterOverrides, b.ko.Spec.ParameterOverrides) {
```

Comment:
Noting this comment from the analogous line in #107:

https://github.com/aws-controllers-k8s/rds-controller/pull/107/files#r948248230